man/legend.grob.Rd

```diff
@@ -118,9 +118,14 @@ legends2 <- list(
         labels = c('Group 1', 'Group 2', 'Group 3'),
         title = expression(underline('Legend #1'))
         ),
-    legend = list(
+    # Use dots instead of rectangles
+    point = list(
         colours = c('firebrick3', 'lightgrey'),
-        labels = c('A label', 'A longer label')
+        labels = c('A label', 'A longer label'),
+        # Set dot size
+        cex = 1.5,
+        # More row spacing needed with dots
+        padding.text = 3
```

Comment:
Is this something that could be changed into a default? Doesn't seem like overlapping text/dots would be a good default setting.